src/Microsoft.ML.PCA/PcaTransform.cs

```diff
@@ -166,7 +257,7 @@ public void Save(ModelSaveContext ctx)
                 ctx.Writer.WriteFloatArray(MeanProjected);
             }
 
-            internal void ProjectMean(Float[] mean)
+            internal void ProjectMean(float[] mean)
```

Comment:
>internal [](start = 12, length = 8)

if it's a private class, you can safely hav epublic methods.

Reply:
This would mean that you'll need to be creative about the loading constructor (it would need to take input and output name as well as `ModelLoadContext`, but it's still good.

---
In reply to: [227068817](https://github.com/dotnet/machinelearning/pull/1333#discussion_r227068817) [](ancestors = 227068817)

Reply:
this was old code. changed it to public.

---
In reply to: [227069529](https://github.com/dotnet/machinelearning/pull/1333#discussion_r227069529) [](ancestors = 227069529,227068817)